#234 - data notes width

hoshmn opened the pull request:
also fix build warnings

hoshmn commented on packages/gatsby-ucla-site/src/components/IconWithTooltip.js:
```diff
@@ -6,16 +6,23 @@ import CloseIcon from "../../content/assets/close-icon.svg"
+  },
   iconWrapper: {
-    marginTop: theme.spacing(2),
+    marginLeft: "auto",
+    marginRight: "auto",
```
Ok to center the trigger element? Otherwise ReactTooltip seems to be having trouble keeping the tooltip from overflowing from the screen (esp on Chrome). Tried futzing with an `offset`, but it gets complicated because the tooltip might go above, below, _or_ right of the trigger element...

<img width="952" alt="Screen Shot 2021-03-18 at 6 04 08 PM" src="https://user-images.githubusercontent.com/18331999/111707021-b52b3600-8819-11eb-8a2e-d985ff9c3d65.png">

hoshmn commented on packages/gatsby-ucla-site/src/components/IconWithTooltip.js:
```diff
@@ -68,11 +78,11 @@ const IconWithTooltip = ({
-    <>
-      <a className={classes.iconWrapper} data-tip data-for={id}>
+    <div className={classes.root}>
+      <button className={classes.iconWrapper} data-tip data-for={id}>
```
<button> rather than <a> for accessibility - was getting the error:
`warning ESLintError: 
/Users/alphalpha/repos/covid-19-behind-bars/packages/gatsby-ucla-site/src/components/IconWithTooltip.js
  72:7  warning  The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid`

Lane commented:
@hoshmn i switched this to material UI components for tooltip, button, typography.  we should stick to the material UI components for this site so that the theme applies correctly.  also removed styles with `!important` in `IconWithTooltip.js`, we should avoid doing that where we can, as it can introduce problems down the line if someone goes to adjust styling on a tooltip.

can you take a quick look at the preview and make sure i didn't mess anything up with these changes?  if it's all good this is good to merge.

hoshmn commented:
@Lane makes sense. For the tooltip I used ReactTooltip since that's what we were using for the map ones, and we weren't using the MUI one anywhere -- but the MUI is simpler, agreed that it makes sense to use. On mobile you have to hold it down for 700ms by default to activate, so I changed to 1. I also removed the arrow, as Sasha had made that request on the initial implementation (he also requested the close button so users have something to aim at, but given that it self-closes after 1.5s or on clicking anywhere it seems ok to remove).
